docs/source/configuration.rst

```diff
@@ -100,7 +100,7 @@ Cluster configuration
 
     Use `preemptible nodes <https://cloud.google.com/kubernetes-engine/docs/how-to/preemptible-vms>`_ in the kubernetes cluster.
 
-    **Note**: This is only recommended if your ElasticBLAST search will take a few hours (less than 24).
+    **Note**: This is only recommended if your ElasticBLAST search will take less than 24 hours.
```

Comment:
IIRC that note is in regards of a search where one BLAST job takes longer than 24 hours, gets pre-empted (as opposed to time out or fail) and keeps getting added back to the queue. It's a bit of a corner case, but theoretically it could happen.

Reply:
I could move this to the known issues and/or I could make clear that this is about a single search.  Will work on it a bit.

Reply:
Did some edits to make this clear.